src/BinCommand.php

```diff
@@ -33,57 +34,85 @@ protected function configure()
      */
     public function execute(InputInterface $input, OutputInterface $output)
     {
-        $this->resetComposers();
+        $this->resetComposers($application = $this->getApplication());
```

Comment:
one avantage of making this `resetComposers()` stateless (i.e. passing the application), is that it can be validated via the typehint